singleheader/simdjson.cpp

```diff
@@ -4951,6 +4954,7 @@ simdjson_warn_unused simdjson_inline error_code tape_builder::visit_root_string(
 
 simdjson_warn_unused simdjson_inline error_code tape_builder::visit_number(json_iterator &iter, const uint8_t *value) noexcept {
   iter.log_value("number");
+  tapeLocToLoc->emplace(tape.next_tape_loc, value);
```

Comment:
This should be optional, so that it comes at no cost to the users who do not need it. E.g., it could be triggered by a template parameter.